src/EditorFeatures/VisualBasicTest/TaskList/VisualBasicTaskListTests.vb

```diff
@@ -4,175 +4,176 @@
 
 Imports Microsoft.CodeAnalysis.Editor.UnitTests.Extensions
 Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
-Imports Microsoft.CodeAnalysis.Test.Utilities.TodoComments
+Imports Microsoft.CodeAnalysis.Remote.Testing
+Imports Microsoft.CodeAnalysis.Test.Utilities.TaskList
 
 Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.TodoComment
     <[UseExportProvider]>
-    Public Class TodoCommentTests
-        Inherits AbstractTodoCommentTests
+    Public Class VisualBasicTaskListTests
+        Inherits AbstractTaskListTests
 
-        Protected Overrides Function CreateWorkspace(codeWithMarker As String) As TestWorkspace
-            Return TestWorkspace.CreateVisualBasic(codeWithMarker)
+        Protected Overrides Function CreateWorkspace(codeWithMarker As String, composition As TestComposition) As TestWorkspace
+            Return TestWorkspace.CreateVisualBasic(codeWithMarker, composition:=composition)
         End Function
 
-        <Fact>
-        Public Async Function TestSingleLineTodoComment_Colon() As Task
+        <Theory, CombinatorialData>
```

Comment:
tests have been updated to ensure that we get the same results, regardless of whether TaskListItems are computed in proc or OOP.